model/src/main/proto/topic.proto

```diff
@@ -249,3 +252,35 @@ message CellularDataPreference {
   // Preference on whether to use cellular data
   bool use_cellular_data = 2;
 }
+
+// Corresponds to a single subtopic.
+message Subtopic {
+  // The ID of the subtopic.
+  string subtopic_id = 1;
+
+  // The title of the subtopic.
+  string title = 2;
+
+  // The list of skill ids associated with the subtopic.
+  repeated string skillIdList = 3;
```

Comment:
Nit: proto names ought to be snake_case, and lists don't need 'list' in the name (otherwise it makes the generated method names a bit odd). I think the recommended practice from the protobuf team is to name repeated items as a plural (we aren't consistent here):

``repeated string skill_ids = 3;``

https://developers.google.com/protocol-buffers/docs/style#repeated-fields

Ditto for other new fields in this PR.

Reply:
Updated.